CMakeLists.txt

```diff
@@ -116,3 +119,10 @@ if (NOT EIGEN3_FOUND)
 	include_directories (${CMAKE_BINARY_DIR}/eigen3-installed/include/eigen3)
 	add_dependencies (opmautodiff Eigen3)
 endif (NOT EIGEN3_FOUND)
+
+
+
+if (HAVE_OPM_DATA)
+   add_test( NAME flow_SPE1 COMMAND flow ${OPM_DATA_ROOT}/spe1/SPE1CASE1.DATA )
```

Comment:
Well - you do not need the XML parameters to run flow with all-default-parameters; so I would say no.